servlet.py

```diff
@@ -212,14 +258,25 @@ def get(self):
                 if self.get_arguments('include_deprecated_codes'):
                     responseData.append({'sourceLanguage': toAlpha2Code(l1), 'targetLanguage': toAlpha2Code(l2)})
             self.sendResponse({'responseData': responseData, 'responseDetails': None, 'responseStatus': 200})
+        elif query == 'chains':
+            src = self.get_argument('src', default=None)
+            if not src:
+                self.send_error(400, explanation='Missing argument src')
+            else:
+                self.sendResponse({
+                    'responseData': list(self.paths[src].keys()),
+                    'responseDetails': None,
+                    'responseStatus': 200
+                })
         elif query == 'analyzers' or query == 'analysers':
             self.sendResponse({pair: modename for (pair, (path, modename)) in self.analyzers.items()})
         elif query == 'generators':
             self.sendResponse({pair: modename for (pair, (path, modename)) in self.generators.items()})
         elif query == 'taggers' or query == 'disambiguators':
             self.sendResponse({pair: modename for (pair, (path, modename)) in self.taggers.items()})
         else:
-            self.send_error(400, explanation='Expecting q argument to be one of analysers, generators, disambiguators or pairs')
+            self.send_error(400, explanation='Expecting q argument to be one of analysers, generators, '
```

Comment:
Looking at this again, perhaps we can combine pairs and chains. Pairs acts normally if there's no "src" and lists the possible destination languages (in the same format) if it does. Thoughts? Just want to have as simple of an API as possible :)